docs/byog.md

```diff
@@ -8,9 +8,15 @@ Such mechanisms are built into the runtime on Picasso and [other Dotsama paracha
 
 `On ramps` and [meta (gas less) transactions](https://docs.polygon.technology/docs/category/meta-transactions) handle such possibilities on other chains.
 
+## Context
+
+Users pay fees in the runtime's native currency for the execution, allowing the runtime to earn and prevents spam and DDoS attacks. BYOG extend payments to registered non native currencies.
+
+A runtime configuration allows mapping amount of execution which can be bought for amount of native currency.
```

Comment:
What does "amount of execution" mean here? Is this referring to the number of computational resources needed to execute?

Reply:
yes.

Reply:
will update in next PR were clarification on price covertion will be made